app/views/help/verbal_ntp_requests.md

```diff
@@ -0,0 +1,5 @@
+<% title "How do I submit an verbal NTP request in case of emergency?" %>
```

Comment:
Nah, same issue with some existing pages and they're our main users right now, so no worries.